src/commands/lambda/functions/uninstrument.ts

```diff
@@ -149,8 +150,12 @@ export const calculateUpdateRequest = (config: LFunctionConfiguration, runtime:
     }
   }
 
-  // Remove AWS_LAMBDA_EXEC_WRAPPER for .NET and Java
-  if (runtimeType === RuntimeType.DOTNET || runtimeType === RuntimeType.JAVA) {
+  // Remove AWS_LAMBDA_EXEC_WRAPPER for .NET and Java or if ASM is enabled
+  if (
+    runtimeType === RuntimeType.DOTNET ||
+    runtimeType === RuntimeType.JAVA ||
```

Comment:
IIUC the overall implementation, this is no longer needed, because the ASM setup has no conditions anymore:
```suggestion
```

Reply:
yep, Jordan is correct. The only 3 cases the wrapper will be added is if the runtime is dotnet, java, or ASM is enabled

Reply:
Ok, my bad, sorry for the noise.

Reply:
don't be sorry! good question for clarification!